pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2251,14 +2261,20 @@ func DomainCreate(status types.DomainStatus) (int, string, error) {
 		// Use rkt tool
 		// get the rkt image hash for this file; if we do not have it in the rkt cache,
 		// convert it
-		// XXX we assume a container has one image. XXX do we check somewhere?
-		ociFilename, err := utils.VerifiedImageFileLocation(status.DiskStatusList[0].ImageSha256)
+		var containerImageSha256 string
+		for _, ds := range status.DiskStatusList {
+			if ds.Format == zconfig.Format_CONTAINER {
+				containerImageSha256 = ds.ImageSha256
+				break
+			}
```

Comment:
This loop is better as a method of domainStatus?

containerImageSha256 = status.ContainerImageSha()